### PR TITLE
New vehicle data SeatOccupancy

### PR DESCRIPTION
**Pull request by atiwari9**
This proposal is to add new vehicle data `SeatOccupancy` to know if a vehicle seat is occupied and belted.

### CODE REVIEW

**Comment by atiwari9** [comment]
@theresalech  - Ready for review

File: proposals/NNNN-New-vehicle-data-SeatOccupancy.md [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+```
+
+## Potential downsides
+
+Author is not aware of any downsides to proposed solution. This proposal just enhances the SDL content.
+
+## Impact on existing code
+
+* SDL Core needs to be updated as per new API.
+* iOS/Java Suite needs to be updated to support getters/setters as per new API.
```
[the remark this answers is not in the document]
Comment (atiwari9) [reply proposals/NNNN-New-vehicle-data-SeatOccupancy.md]: Done

File: proposals/NNNN-New-vehicle-data-SeatOccupancy.md [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+* Status: **Awaiting review**
+* Impacted Platforms: [Core | HMI | Policy Server | SHAID | iOS | Java Suite | RPC]
+
+## Introduction
+
+This proposal is to add new vehicle data `SeatOccupancy` to know if a vehicle seat is occupied and belted.
+
+## Motivation
+
+In order to partner with more diverse app partners, we need to provide additional sets of vehicle data items through SDL. This goes in line with commitment to enhance SDL with even richer vehicle data content. For insurance apps and emergency assistance apps, it is important to know what all seats are occupied and belted.
```
[the remark this answers is not in the document]
Comment (atiwari9) [reply proposals/NNNN-New-vehicle-data-SeatOccupancy.md]: Done

**Comment by atiwari9** [comment]
@theresalech  - I have updated the proposal as per your review comments.

**Comment by atiwari9** [comment]
@theresalech - I have pushed another commit to implement grids for this proposal as well. Can you please review this again when you get time.

File: proposals/NNNN-New-vehicle-data-SeatOccupancy.md [hunk 3]
```diff
@@ -25,64 +25,26 @@ We need to add `SeatOccupancy` for `GetVehicleData`, `SubscribeVehicleData`, `Un
 ```xml
 <element name="VEHICLEDATA_SEATOCCUPANCY" since="X.x"/>
 ```
+
+#### Add new struct `SeatStatus`:
+```xml
+<struct name="SeatStatus" since="X.x">
+	<description>Describes the Status of a parameter of seat.</description>
```
Comment (theresalech) [inline proposals/NNNN-New-vehicle-data-SeatOccupancy.md]: Should "Status" be lowercase?

File: proposals/NNNN-New-vehicle-data-SeatOccupancy.md [hunk 4]
```diff
@@ -124,66 +86,30 @@ We need to add `SeatOccupancy` for `GetVehicleData`, `SubscribeVehicleData`, `Un
 ```xml
 <element name="VEHICLEDATA_SEATOCCUPANCY"/>
 ```
+
+#### Add new struct `SeatStatus` in `Common` interface:
+
+```xml
+<struct name="SeatStatus">
+	<description>Describes the Status of a parameter of seat.</description>
```
Comment (theresalech) [inline proposals/NNNN-New-vehicle-data-SeatOccupancy.md]: Should "Status" be lowercase?

Comment (atiwari9) [reply proposals/NNNN-New-vehicle-data-SeatOccupancy.md]: Done

File: proposals/NNNN-New-vehicle-data-SeatOccupancy.md (hunk 3 above)
Comment (atiwari9) [reply proposals/NNNN-New-vehicle-data-SeatOccupancy.md]: Done

**Comment by atiwari9** [comment]
@theresalech  - I have updated the proposal with your review comments.

**Pull request merged**